core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -102,6 +115,11 @@ private void initializeCamera() {
     }
 
     public void drawStartScreen() {
+        if(!backgroundSound.isPlaying()){
```

Comment:
there is  no need to check this every frame. Pls move this to an own method and only call it within the create() method.

Reply:
Created an own method to start music, which is once called after the initialization.
Maybe the if-statement is unnecessary at this point. Please check if it's useful for protective reasons to prevent playing same music in multiple loops.